os/include/tinyara/sched.h

```diff
@@ -500,7 +494,12 @@ struct task_group_s {
 #endif
 
 #ifdef CONFIG_BINFMT_LOADABLE
-#define IS_LOADED_MODULE(group)    (group->tg_bininfo != NULL)   /* Points loading data if it is loaded */
+/* This macro verifies the tcb is for a main task of binary.
+ * A main task of binary has loading data, bininfo only.
+ * So it checks whether a tcb is for task and has binary loading data or not.
+ */
```

Comment:
Could you modify this comment a bit please?
`This macro verifies whether the tcb is for a main task of the binary.
It checks if the tcb is for a task and if it has non NULL loading data i.e. 'bininfo' `